kubernetes/charts/weblogic-operator/templates/_operator-internal-svc.tpl

```diff
@@ -17,4 +17,19 @@ spec:
   ports:
     - port: 8082
       name: "rest"
+---
+apiVersion: "v1"
+kind: "Service"
+metadata:
+  name: "internal-weblogic-operator-webhook-svc"
```

Comment:
Why "internal"? The operator has two different services with the external one meant to possibly be accessed from out side the cluster, but the webhook will only ever be accessed from the k8s master. I think you can just remove "internal-"

Reply:
Removed "internal-".